docs/fundamentals/wadm/deploying.md

```diff
@@ -2,13 +2,211 @@
 title: "Deploying Wadm"
 date: 2020-01-19T00:00:00+00:00
 icon: "ti-map" # themify icon pack : https://themify.me/themify-icons
-description: "Deploying and Running the Wadm Application in a Cluster"
+description: "Deploying and Running the Wadm Application"
 type: "docs"
 sidebar_position: 1
 ---
 
-This document will guide you through the steps and configuration necessary to deploy the [wadm](https://github.com/wasmcloud/wadm) application in a lattice.
+This guide will walk you through the steps to deploy the [wadm](https://github.com/wasmcloud/wadm)
+application in a lattice. We will cover how to deploy wadm from binaries or Docker, and how to
+configure NATS credentials for controlling access to wadm
 
-:::info
-This document is under construction as **[wadm](https://github.com/wasmcloud/wadm)** undergoes a transition to the new Rust target. Until then, see the GitHub repository for the most current information.
-:::
\ No newline at end of file
+## Installing Wadm
+
+We offer wadm as a standalone binary or as a docker image.
+
+### Standalone Binary
+
+You can download the latest release of wadm from the [releases
+page](https://github.com/wasmCloud/wadm/releases/latest). We have the following binaries available:
+
+- Linux (x86_64)
+- Linux (aarch64)
+- MacOS (x86_64)
+- MacOS (aarch64)
+- Windows
+
+In the future we will likely offer easier ways to install wadm (such as apt, homebrew, chocolatey,
+etc), but for now you'll have download the binary.
```

Comment:
IMO for things we _think_ we'll do, but haven't done yet, we should have a prompt for feedback instead: "if you want to install wadm via a package manager, we'd love to hear your feedback. File an issue in the repo"

I'd grep for "future", "hope", "soon" and consider removing that language and replacing with CTAs if we want to test the hypothesis

Reply:
I actually did think about doing this so this is confirmation that I should have done it. Will update

Reply:
Let me know what you think about the changes